src/main/java/com/example/bulletinboard/controller/UserController.java

```diff
@@ -0,0 +1,44 @@
+package com.example.bulletinboard.controller;
+
+import com.example.bulletinboard.dto.user.NewPassword;
+import com.example.bulletinboard.dto.user.UpdateUser;
+import com.example.bulletinboard.dto.user.UserDto;
+import com.example.bulletinboard.service.UserService;
+import io.swagger.annotations.Api;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+
+@Api(tags = "Пользователи")
+@RestController
+@RequestMapping("/users")
+@CrossOrigin(value = "http://localhost:3000")
+@RequiredArgsConstructor
+public class UserController {
+
+    private final UserService userService;
+
+    @PostMapping("/setPassword")
```

Comment:
```suggestion
    @PostMapping("/setPassword")
```
```suggestion
    @PostMapping("/set_password")
```
здесь вот такой путь